trl/trainer/ppo_trainer.py

```diff
@@ -229,12 +230,17 @@ def __init__(
         elif ref_model is None and not self.is_peft_model:
             self.ref_model = create_reference_model(self.model, num_shared_layers=num_shared_layers)
         elif self.is_peft_model:
-            self.ref_model = None
+            self.ref_model = self.model
```

Comment:
this is breaking no? Maybe some workflows use the check `if ppo_trainer.ref_model is None` . Do you think there is a way to achieve what you want to do in the PR without modifying this condition? 🙏 
What do you think?

Reply:
Thanks for the commnet, I have addressed it by creating a separate var